transmart-api-server/grails-app/init/org/transmart/api/server/Application.groovy

```diff
@@ -0,0 +1,12 @@
+package org.transmart.api.server
+
+import grails.boot.GrailsApp
+import grails.boot.config.GrailsAutoConfiguration
+import org.springframework.boot.autoconfigure.security.oauth2.client.EnableOAuth2Sso
+
+@EnableOAuth2Sso
+class Application extends GrailsAutoConfiguration {
+    static void main(String[] args) {
+        GrailsApp.run(Application, args)
+    }
```

Comment:
Doesn't this require the `setEnvironment()` function to read the external config files? 
```
    @Override
    void setEnvironment(Environment environment) {
        if (grails.util.Environment.current == grails.util.Environment.TEST) {
            return
        }

        String applicationName = 'transmart'
        String userHome = System.getProperty('user.home').replace('\\','/')

        addToEnvironmentAsFirstSource(environment, "${userHome}/.grails/${applicationName}Config/Config.groovy")
        addToEnvironmentAsFirstSource(environment, "${userHome}/.grails/${applicationName}Config/DataSource.groovy")
        addToEnvironmentAsFirstSource(environment, "${userHome}/.grails/${applicationName}Config/application.groovy")
(...)
```

Reply:
@ewelinagr No, apparently there is another, imho more cleaner, way to specify an external configuration file with smth like 'java --spring.config.location ...'.
Otherwise the internal configuration file will be used.
Please see https://jira.thehyve.nl/browse/TMT-229